build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
 
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.0'
+        classpath 'com.android.tools.build:gradle:4.0.1'
```

Comment:
Need to update your gradle to 4.0.1.

Reply:
See note above on upgrading minor versions.